<launch>
    <arg name="drone_id" default="1"/>
    <!-- Launch mission_planner_node -->
    <node pkg="mission_planner" type="mission_planner_node" name="mission_planner_ros" output="screen" ns="$(arg drone_id)" />

</launch>
